src/main/scala/edu/berkeley/ce/rockslicing/InputProcessor.scala

```diff
@@ -1,16 +1,15 @@
 package edu.berkeley.ce.rockslicing
 
-import breeze.linalg.DenseVector
-
 import scala.io.Source
 import scala.util.Try
 
 object InputProcessor {
   // Processes input file: Add rock volume faces and joints to respective input list
-  def readInput(inputSource: Source): Option[(Array[Double],
-                                             Seq[Face], Seq[Joint])] = {
+  def readInput(inputSource: Source): Option[ (Array[Double], Array[Double],
+    Vector[Array[Double]], Vector[Array[Double]]) ] = {
```

Comment:
Can we use `Seq` here?

The caller of this function shouldn't care if they get back a `Vector` specifically.